iamproperty/iamproperty.github.io
# Ds 50/header

## Summary of Changes:
1. Documentation header component
2. Header Vue component
3. Unit tests

## Rationale
Document the header component and how to implement it.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/header

## Ticket(s)
DS-50

## Checklist

## Notes
-
